content/en/docs/concepts/services-networking/service.md

```diff
@@ -384,6 +384,40 @@ The IP address that you choose must be a valid IPv4 or IPv6 address from within
 If you try to create a Service with an invalid clusterIP address value, the API
 server will return a 422 HTTP status code to indicate that there's a problem.
 
+## Traffic policies
+
+### External traffic policy
```

Comment:
should we mention the source IP preservation, `Local` preserves the original IP, however `Cluster`masquerades the original IP with the Kubernetes node IP.

Reply:
nah, it is documented in other places in the doc and in the web `content/en/docs/tutorials/services/source-ip.md`